### ISSUE TITLE
Add lapack link options to make travis work

### ISSUE THREAD

**Issue by francescosalvadore** [francescosalvadore/openpde#9]
Using intel compiler with mkl loaded in the environment this compilations works
FoBiS.py build -mode tests-intel-debug -cflags="-mkl" -lflags="-mkl"
but travis compiles using gfortran and does not work now. To be fixed.

**Comment by szaghi** [comment]
@francescosalvadore using Intel on Travis is possible, but not simple and the license terms is not clear to me. In my starred repo-list I found one repo devoted to Intel compilers on Travis, but with only my dated smartphone I cannot link it.

**Comment by francescosalvadore** [comment]
@szaghi probably the best is just to add https://github.com/qsnake/lapack as submodule and setup the compilation using it. The linear solver used so far is just for testing since it uses very slow general matrixes.

**Comment by szaghi** [comment]
@francescosalvadore ok, tomorrow I will be st office (for a while) I can do it, but you have to push :smile:

**Comment by szaghi** [comment]
@francescosalvadore 

### other linear solvers implementation
- https://github.com/Reference-LAPACK/lapack
- https://github.com/xianyi/OpenBLAS

**Comment by francescosalvadore** [comment]
@szaghi I don't know. If it compiles and works it is ok with me...

**Comment by szaghi** [comment]
Yes, I will play with some alternatives later today... I hope to push one reasonable solution. My concern is that lapack is about 80MB of data, I do not like to add it as git submodule...
